app/controllers/api/subcollections/cloud_tenants.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module Subcollections
+    module CloudTenants
+      def cloud_tenants_query_resource(object)
+        object.respond_to?(:cloud_tenants) ? object.cloud_tenants : []
```

Comment:
I think if it's anticipating a need for it later, it would be better in the simpler form for now. If you can prove that it's needed in a later revision (through testing, which you'd need to provide anyway) you can always change that when it's needed. Thanks!